include/mbedtls/mbedtls_config.h

```diff
@@ -328,6 +328,7 @@
 //#define MBEDTLS_SHA1_ALT
 //#define MBEDTLS_SHA256_ALT
 //#define MBEDTLS_SHA512_ALT
+//#define MBEDTLS_SHA3_ALT
```

Comment:
Adding `MBEDTLS_SHA3_ALT`, while thoughtful, isn't really the direction we're heading in now - we'd rather not add any more of these for crypto algorithms. In the future, replaceable implementations will use the PSA driver interface, so unless there's a pressing need for this right now, it would be better removed.